models/runner/README.md

```diff
@@ -46,6 +46,12 @@ Compress-Archive -Path .\models\runner_win\, .\models\gabon\, .\models\serengeti
 
 ## Running in dev mode directly
 
+These instructions were tested on Debian 9.13 with Python 3.6.9.
+You need to install the following system dependencies:
+```sh
+sudo apt install libmkl_rt libtorch-dev
+```
+
 1. Create a virtual environment (`python3 -m venv venv`)
```

Comment:
I couldn't install dependencies from step 2, until as step 1 I entered `python -m venv venv` instead of python3. Then the dependencies got installed. `python3 -V` was returning `Python 3.8.1` (installed with ubuntu), `python -V` was returning `Python 3.6.9`